pkg/actors/internal_actor.go

```diff
@@ -30,6 +31,7 @@ import (
 )
 
 const InternalActorTypePrefix = "dapr.internal."
+const InternalActorIdleTimeout = time.Minute * 1
```

Comment:
With #6538 we will have more internal actors, so hardcoding a timeout may not work for all cases.

WHat about modifying `RegisterInternalActor` so the internal actor can define its own idle timeout?

Reply:
Modified accordingly, had to store idle timeout against actor type, so have used `hostedActors` struct for that